app/scripts/lib/metrics.js

```diff
@@ -152,7 +152,7 @@ define([
       var url = this._collector + '/metrics';
       var self = this;
 
-      return this._send(filteredData, url)
+      return this._send(filteredData, url, ! flushSynchronously)
```

Comment:
> * If a flush has already occurred and filteredData.events, filteredData.timers, and filteredData.ab are all empty, then we have no new information; pinging the metrics endpoint doesn't really make sense. I wonder if this could ever occur on the first send.

@shane-tomlinson, I am seeing this happen quite frequently locally; I think, as a result of the new `blur` behaviour.

I have a fix locally, checking whether `events` and `timers` are populated, but I am confused by your mention of `filteredData.ab`. I don't see it get emptied like the other two. Should I instead be checking to see whether it has changed since the last invocation, or am I missing something?